src/gen_dragonfruit/gd_dev_xbee.cpp

```diff
@@ -6,6 +6,16 @@ void gd_dev_xbee_open(void)
     xbee.begin(soft_serial);
     // Enable voltage regulator pin to power the Xbee 
     digitalWrite(3, HIGH);
+
+    // Enable DTR pin as output
```

Comment:
The descriptions for these pin configurations could be a little more precise. The phrase `Enable DTR pin as output` implies that there is a DTR pin on the MCU itself.

It's in the same style as the comment above, but I think it could probably be improved a little to provide a more accurate description. Something like this would probably work:

```
\\ Configure pin connected to DTR on XBee
\\ Since in router mode we don't care what the output is we can drive this signal LOW by default.
```